doc/rnc/lifecycle_configuration.rnc

```diff
@@ -24,11 +24,21 @@ Expiration =
   element ExpiredObjectDeleteMarker { xsd:boolean }
 
 Filter =
-  element Prefix { xsd:string }? &
-  element Tag {
-    element Key { xsd:string } &
-    element Value { xsd:string }
-  }*
+  (
+    element Prefix { xsd:string }? &
+    element Tag {
+      element Key { xsd:string } &
+      element Value { xsd:string }
+    }
+  ) | (
+    element And {
+      element Prefix { xsd:string }? &
+      element Tag {
+        element Key { xsd:string } &
+        element Value { xsd:string }
+      }*
```

Comment:
We could define `Prefix` and `Tag` only once to factorize this schema.

Reply:
I did not define Prefix since it is just a string, but I defined Tag.